src/subcommands/create_rsa_key.rs

```diff
@@ -27,6 +27,11 @@ pub struct CreateRsaKey {
     #[structopt(short = 's', long = "for-signing")]
     is_for_signing: bool,
 
+    /// This command creates RSA encryption keys by default. Supply this flag to create a signing key instead.
+    /// Signing keys, by default, will specify the SHA-256 hash algorithm and use PKCS#1 v1.5.
```

Comment:
Perhaps we should change the comment here to state that this is for PSS? Kind of obvious, but just from the comment one can not discern what is the difference between -s and -p

Reply:
Yes, this is not updated. Will make the change.